lib/process-ctl.js

```diff
@@ -86,9 +86,17 @@ ProcessCtl.prototype.exec = function(cmd, options) {
   log.info('executing: ' + cmd);
 
   var self = this;
-  this.process = childProcess.exec(cmd, _options, function(err, stdout, stderr) {
-    self.stdout = stdout;
-    self.stderr = stderr;
+  var cmdParts = cmd.split(" ");
```

Comment:
This won't support quoted commands like: `node -e "console.log('with spaces');"`